app/views/mod.py

```diff
@@ -0,0 +1,225 @@
+""" Mod endpoints """
+import time
+import re
+from peewee import fn, JOIN, Value
+from pyotp import TOTP
+from flask import Blueprint, abort, redirect, url_for, session, render_template, jsonify
+from flask_login import login_required, current_user
+from flask_babel import _
+from .. import misc
+from ..forms import TOTPForm, LogOutForm, UseInviteCodeForm, AssignUserBadgeForm, EditModForm, BanDomainForm
+from ..models import UserMetadata, User, Sub, SubPost, SubPostComment, SiteMetadata
+from ..models import User, Sub, SubMod, SubPost, SubPostComment, UserMetadata, SubPostReport, SubPostCommentReport
+from ..misc import engine, getSubReports, getModSubs, engine
+
+bp = Blueprint('mod', __name__)
+
+
+@bp.route('/mod/auth', methods=['GET', 'POST'])
+@login_required
+def auth():
```

Comment:
I don't see any redirects to this endpoint. Also our current TOTP implementation currently only works for admin pages (and the secrets have to be manually inserted into the database...)

If you plan on implementing TOTP for everybody we should do it in a different pull and do a more broad implementation (covering the account settings page, sub settings, etc)

Reply:
nah you're right - I was just blindly copying the Admin view at first :)